src/Utilities/generateGitHubMarkdown.ts

```diff
@@ -27,14 +27,14 @@ export function generateGitHubMarkdown(
 
         markdownArray.push('\n');
         markdownArray.push(`### ${dependency}`);
-        markdownArray.push('|**Title**|**Age**|');
+        markdownArray.push('|**Title**|**Age**|**Languages**|');
```

Comment:
I'm not sure if README changes get picked up in this review. You may consider also adding the `Languages` column to the example output in the README -- see: https://github.com/indeedeng/mariner#optional-generating-markup

Reply:
Thank you, I will add it